pyartcd/pyartcd/pipelines/build_sync.py

```
@@ -434,6 +450,121 @@ async def _populate_ci_imagestreams(self):
+
+            tasks = []
+            for tag in tags_to_transfer:
+                # Mirror to public QCI
+                tasks.append(self._mirror_single_rhcos_to_qci(tag, prune_timestamp, private=False))
+                # Mirror to private QCI (for openshift-priv CI)
+                tasks.append(self._mirror_single_rhcos_to_qci(tag, prune_timestamp, private=True))
+
+            await asyncio.gather(*tasks)
+            self.logger.info('Successfully mirrored RHCOS images to QCI')
+
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)